tic-tac-toe.js

```diff
@@ -1,7 +1,91 @@
-function TicTacToe() {
+$(document).ready(function () {
+  var game = new TicTacToe();
+  var cells = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 
-}
+  cells.forEach(function(cell) {
+    $square = $("#cell" + cell);
+    $square.one( "click", (function () {
+      $(this).css("background-color", "#16A765");
+       var id = $(this).attr("id");
+      if (game.turns % 2 === 0) {
+        $(this).css("background-color", "#000");
+      }
+      game.play(id);
+    }));
+  });
+});
 
-TicTacToe.prototype = {
-  
+function TicTacToe() {
+  this.player = "Player 1";
+  this.turns = 1;
+  this.board = [[0, 1, 2], [3, 4, 5], [6, 7, 8]];
+  this.finish = false;
 }
+
+TicTacToe.prototype.turn = function(id) {
+  var block = id[4] - 1;
+  var row;
+  if (block < 3) {
+    row = 0;
+  } else if (block >= 3 && block < 6) {
+    row = 1;
+  } else {
+    row = 2;
+  }
+  column = block % 3;
+
+  if (this.player == "Player 1") {
+    this.board[row][column] = "X";
+  } else {
+    this.board[row][column] = "O";
+  }
+  // checks turn and assigns player
+  this.turns++;
+  if (this.turns % 2 === 0) {
+    this.player = "Player 2";
+  } else {
+    this.player = "Player 1";
+  }
+};
+
+TicTacToe.prototype.play = function(id) {
+  this.turn(id);
+  this.win();
+  if (this.finish === false) {
+    if (this.turns === 10) {
+      $("#notice").text("It's a draw!");
+    }
+  } else {
+      $("#notice").text(this.player + " wins!");
+    }
+  };
+
+
+TicTacToe.prototype.win = function() {
+  // rows
+  for (var i = 0; i < this.board.length; i++) {
+    console.log(this.board[0][i]);
```

Comment:
Based on how your board is initialized, this will give you only the values 1,2,3. I think you want to check 1,2,3 against each other as well as 4,5,6 as well as 7,8,9.
You can do this using a loop, like you have, but you'd want to check the whole set of rows instead of each individual row. 
Something like
```
var row = this.board[i]; // [1,2,3] or [4,5,6] or [7,8,9]
// then you could do you if statement because row[0] will equal 1, row[1] will equal 2, etc.
```